main.py

```
@@ -1,11 +1,273 @@
+from bs4 import BeautifulSoup
+from typing import Tuple
+
+from download_controller import DownloadControllerAsync, DownloadServiceAsync
+from reader_controller import ReadingController
+from rss_controller import RSSController
+from rss_repository import RSSRepository
+
+class ClickableComicWidget(QWidget):
+    clicked = Signal()
+
+    def __init__(self, title: str, pixmap: QPixmap, img_width = 20, img_height=20, parent=None):
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)